src/main/java/pl/widulinski/scrapp/webDriver/DriverBuilder.java

```diff
@@ -0,0 +1,24 @@
+package pl.widulinski.scrapp.webDriver;
+
+
+import lombok.Data;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+@Data
+public abstract class DriverBuilder {
```

Comment:
Czemu to jest klasa abstrakcyjna. Zarówno nazwa jak i to co robi brzmi na idealnego kandydata na beana (`@Component`) który bedzie wstrzykiwany wszedzie gdzie aktualnie jest "rodzicem". 
Czemu to zły kandydat na rodzica? Proste pytanie - co ma wspolnego `DriverBuilder` i `UrlService` - nic, ani z nazwy ani z funkcjonalności. Więc tworzenie sztucznej hierarchi miedzy nimi nie ma sensu.
Poza tym nazwa `builder` to słaba nazwa. Klasa która jest builderem to zazwyczaj rozległa klasa z masą setterów - poczytaj o tym wzorcu. Tutaj to bardziej `Factory` albo `Service` po prostu

Reply:
Url service korzysta z drivera przy skrobaniu więc myślałem, że muszą być ściśle powiązane ze sobą. Zamysł był taki aby stworzyć osobną klasę w której będzie tworzony driver wykorzystujący przeglądarkę wybraną przez użytkownika. OK zmieniam nazwę na "BrowserPicker"